grunttasks/jscs.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('jscs', {
+    app: {
+      src: [
+        '<%= eslint.files %>'
+      ],
+      options: {
+        config: '.jscsrc',
```

Comment:
Ugh, it was being ignored by the .gitignore so didn't get checked in, I've added it now